python-package/setup.py

```diff
@@ -322,7 +322,7 @@ def run(self):
     if os.path.isfile(os.path.join(CURRENT_DIR, os.path.pardir, 'VERSION.txt')):
         copy_file(os.path.join(CURRENT_DIR, os.path.pardir, 'VERSION.txt'),
                   os.path.join(CURRENT_DIR, 'lightgbm', 'VERSION.txt'),
-                  verbose=0)
+                  verbose=False)
```

Comment:
Was this to fix this warning?

> python-package/setup.py:325: error: Argument "verbose" to "copy_file" has incompatible type "int"; expected "bool"

I think this is another case where `mypy` is getting confused. I can see in the definition of `copy_files()` that verbose is an int: https://github.com/python/cpython/blob/9eb11a139fac5514d8456626806a68b3e3b7eafb/Lib/distutils/file_util.py#L68.

So can you please add a `# type: ignore` comment here to tell `mypy` to trust us on this line?

```suggestion
                  verbose=0)  # type:ignore
```